css/index.css

```
@@ -0,0 +1,355 @@
+        :root {
+            /* body color */
+            --body: #CFE5FF;
+
+            /* primary color */
+            --primary_100: #3692FF;
+            --primary_200: #1967D6;
+            --primary_300: #1251AA;
+
+            /* error color */
+            --error: #F74747;
+
+            /* Secondary color */
+            --gray_50: #F9FAFB;
+            --gray_100: #F3F4F6;
+            --gray_200: #E5E7EB;
+            --gray_400: #9CA3AF;
+            --gray_500: #6B7880;
+            --gray_600: #4B5563;
+            --gray_700: #374151;
+            --gray_800: #1F2937;
+            --gray_900: #111827;
+        }
+
+        body {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+            font-family: 'pretendard', 'Abhaya Libre ExtraBold' -apple-system, BlinkMacSystemFont, system-ui, Roboto,
+                'Helvetica Neue', 'Segoe UI', 'Apple SD Gothic Neo', 'Noto Sans KR', 'Malgun Gothic',
+                sans-serif;
+            background-color: var(--body);
+        }
+
+        a {
+            text-decoration: none;
+        }
+
+        /* header */
+
+        header {
+            height: 70px;
+            background-color: #ffffff;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            position: sticky;
+            top: 0;
+            z-index: 3;
+        }
+
+        header .login {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            width: 60%;
```

❗️ 수정요청
width가 60%인 이유가 뭘까요? 요구사항을 구현하려면 width가 100%여야 할 것 같아요!


```suggestion
            width: 100%;
```